.ci/Jenkinsfile

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env groovy
```

Comment:
Just curious, this is the new way to specify jenkins job on infra team's end, right?

Reply:
Yes, they monitor `.ci` dirs in Elastic repos, so they are able to _build_ a Jenkins instance based on this meta data. The YAML code that generates this build (https://apm-ci.elastic.co/job/code/) on our CI (it will eventually be moved to yours) is here: https://github.com/elastic/apm-pipeline-library/blob/master/.ci/jobs/code-simple-git.yml (which will be eventually moved too. It's located here to avoid noise in your repo)